### PR TITLE
Release notes draft for 7.0.1

### PR DESCRIPTION
**Pull request by logstashmachine**

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -3,13 +3,89 @@
...
+   the pipeline. By default, timeout is a generous 30s, but can be configured or disabled entirely with the new
+   `timeout_millis` and `tag_on_timeout` directives (https://github.com/logstash-plugins/logstash-filter-kv/pull/79[#79])
+* Made error-handling configurable with `tag_on_failure` directive.
+
+*Xml Filter*
+
+* Fixed creation of empty arrays when xpath failed https://github.com/logstash-plugins/logstash-filter-xml/pull/59[#59]
+
+*Uuid Filter*
+
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: No content for this plugin.

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -3,13 +3,89 @@
...
+*Dead_letter_queue Input*
+
+* Fixed asciidoc formatting in documentation https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/21[#21]
+
+*Couchdb_changes Input*
+
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: No content for this plugin.

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -3,13 +3,89 @@
...
+* Added a timeout enforcer which prevents inputs that are pathological against the generated parser from blocking
+   the pipeline. By default, timeout is a generous 30s, but can be configured or disabled entirely with the new
+   `timeout_millis` and `tag_on_timeout` directives (https://github.com/logstash-plugins/logstash-filter-kv/pull/79[#79])
+* Made error-handling configurable with `tag_on_failure` directive.
+
+*Xml Filter*
+
+* Fixed creation of empty arrays when xpath failed https://github.com/logstash-plugins/logstash-filter-xml/pull/59[#59]
+
+*Uuid Filter*
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc]:
```suggestion
```

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -3,13 +3,89 @@
...
+
+*Dead_letter_queue Input*
+
+* Fixed asciidoc formatting in documentation https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/21[#21]
+
+*Couchdb_changes Input*
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc]:
```suggestion
```

**Pull request merged**